data/pigui/modules/settings-window.lua

```diff
@@ -490,10 +464,25 @@ local function showControlsOptions()
 	c,joystickEnabled = checkbox(lui.ENABLE_JOYSTICK, joystickEnabled)
 	if c then Input.SetJoystickEnabled(joystickEnabled) end
 
+	-- list all the joysticks
+	local joystick_count = Input.GetJoystickCount()
+	if joystick_count > 0 then
+		ui.separator()
+		ui.text(linput.JOYSTICKS .. ":")
+		for id = 0, joystick_count - 1 do
+			ui.text("  " .. bindManager.joyAcronym(id) .. ":")
+			ui.sameLine()
+			ui.withStyleColors({["Text"] = ui.theme.colors.grey}, function()
+				local status = Input.IsJoystickConnected(id) and ", " .. linput.CONNECTED or ", " .. linput.NOT_CONNECTED
```

Comment:
Instead of having the string concatenation on this line, I'd move it to the next line as part of `.. status`.

Reply:
Done!